tests/linalg_test.py

```diff
@@ -577,6 +577,22 @@ def args_maker():
                             check_dtypes=True, tol=1e-3)
     self._CompileAndCheck(np.linalg.inv, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+       "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype, "rng_factory": rng_factory}
+      for shape in [(1, 1), (4, 4), (2, 70, 7), (int(7e5), 70), (70, int(7e6)), (70, 7, 2)]
```

Comment:
Some of these arrays are too big for testing purposes. 7e5 x 70 elements is about 400 MB in complex128 and 70 x 7e5 elements is 10x larger! This is causing the tests on Travis-CI to fail.

We definitely don't want to require this much free memory to test JAX. I would suggest sticking to a maximum array size of no more than 1e6 or so total elements. But  even that is likely bigger than  we really need.

Given  that  this test does not implement any novel numerical methods, it doesn't need to test numerics  exhaustively. That's what XLA's own unit tests are for.